app/controllers/hyku_addons/account_settings_controller.rb

```diff
@@ -35,15 +35,9 @@ def update_single
 
       def account_params
         params.require(:account).permit(
-          settings: [:contact_email, :index_record_to_shared_search, :live, :enabled_doi, :gtm_id,
-                     :turn_off_fedora_collection_work_association, :file_size_limit, :enable_oai_metadata,
-                     :add_collection_list_form_display, :hide_form_relationship_tab, :shared_login,
-                     :redirect_on, :institutional_relationship_picklist, :institutional_relationship,
-                     :oai_prefix, :oai_sample_identifier, :oai_admin_email, :allow_signup,
-                     email_format: [], help_texts: {}, work_unwanted_fields: {}, work_type_list: [],
-                     required_json_property: {}, contributor_roles: [], metadata_labels: {},
-                     creator_roles: [], html_required: {}, licence_list: [:name, :url],
-                     weekly_email_list: [], monthly_email_list: [], yearly_email_list: []]
+          settings: [:contact_email, :gtm_id, :file_size_limit, :enable_oai_metadata,
```

Comment:
I thought we were going to get rid of `:enable_oai_metadata` because it is going into a feature flipper.

Reply:
@cjcolvar its not on the spreadsheet I was working from. I can remove it though.